docs/installation/sources/sources_windows.rst

```diff
@@ -0,0 +1,376 @@
+  command:
+
+  .. code-block:: bash
+
+      setup.bat
+
+* Add the sourcing of the colcon overlay permanently, by opening the
+  *Edit the system environment variables* control panel, and adding :code:`~/Fast-DDS/install/setup.bat`
+  to the :code:`PATH`.
+
+
+.. _cmake_installation_windows:
+
+CMake installation
+-------------------
+
+This section explains how to compile *eProsima Fast DDS* with CMake_, either :ref:`locally <local_installation_sw>` or
+:ref:`globally <global_installation_sw>`.
+
+.. _local_installation_sw:
+
+Local installation
+^^^^^^^^^^^^^^^^^^
+
+#. Open a command prompt, and create a :code:`Fast-DDS` directory where to download and build *eProsima Fast DDS* and
+   its dependencies:
+
+   .. code-block:: bash
+
+       mkdir ~/Fast-DDS
+
+#. Clone the following dependencies and compile them using CMake_.
+
+   * `Foonathan memory <https://github.com/foonathan/memory>`_
+
+     .. code-block:: bash
+
+         cd ~/Fast-DDS
+         git clone https://github.com/eProsima/foonathan_memory_vendor.git
+         cd foonathan_memory_vendor
+         mkdir build && cd build
+         cmake .. -DCMAKE_INSTALL_PREFIX=~/Fast-DDS/install -DBUILD_SHARED_LIBS=ON
+         cmake --build . --target install
+
+   * `Fast CDR <https://github.com/eProsima/Fast-CDR.git>`_
+
+     .. code-block:: bash
+
+         cd ~/Fast-DDS
+         git clone https://github.com/eProsima/Fast-CDR.git
+         cd Fast-CDR
+         mkdir build && cd build
+         cmake .. -DCMAKE_INSTALL_PREFIX=~/Fast-DDS/install
+         cmake --build . --target install
+
+#. Once all dependencies are installed, install *eProsima Fast DDS*:
+
+   .. code-block:: bash
+
+       cd ~/Fast-DDS
+       git clone https://github.com/eProsima/Fast-RTPS.git
+       cd Fast-RTPS
+       mkdir build && cd build
+       cmake ..  -DCMAKE_INSTALL_PREFIX=~/Fast-DDS/install -DCMAKE_PREFIX_PATH=~/Fast-DDS/install
+       cmake --build . --target install
+
+.. _global_installation_sw:
+
+Global installation
+^^^^^^^^^^^^^^^^^^^
+
+To install *eProsima Fast DDS* system-wide instead of locally, remove all the flags that
+appear in the configuration steps of :code:`Fast-CDR` and :code:`Fast-RTPS`, and change the first in the
+configuration step of :code:`foonathan_memory_vendor` to the following:
+
+.. code-block:: bash
+
+    -DCMAKE_INSTALL_PREFIX=??
+
+.. note::
+
+    By default, *eProsima Fast DDS* does not compile tests.
+    However, they can be activated by downloading and installing `Gtest <https://github.com/google/googletest>`_.
+
+.. _run_app_cmake_sw:
+
+Run an application
+^^^^^^^^^^^^^^^^^^
+
+When running an instance of an application using *eProsima Fast DDS*, it must be linked with the library where the
+packages have been installed. There are two possibilities:
+
+* Prepare the environment locally by typing the command:
+
+  .. code-block:: bash
+
+      C:\> PATH=~/Fast-DDS/install/bin
```

Comment:
In the system-wide case, you'd need to add the above directories to the `PATH`, but just the ones for Fast CDR and Fast DDS